Orange/widgets/data/owselectrows.py

```diff
@@ -144,6 +145,13 @@ def add_row(self, attr=None, condition_type=None, condition_value=None):
         attr_combo.setCurrentIndex(attr or 0)
         self.cond_list.setCellWidget(row, 0, attr_combo)
 
+        index = QPersistentModelIndex(model.index(row, 3))
+        temp_button = QPushButton(u'\u00d7', self, flat=True,
```

Comment:
There is no reason to prefix string literals with `u''`in Python 3 since all strings are already unicode. Bikesheddingly, a literal '×' would also be just fine since it's not a very exotic code point.